### PR TITLE
add security information

### PR DESCRIPTION
**Pull request by marcusvbrangel**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Just added security information line..

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [x] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <h1>Super Contact Modified...</h1>
 <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
 
+Adding security information....
+
```
Comment (dijobe) [inline contact.html:5]: please, could you put a style bold on this line????

**Review by dijobe — CHANGES_REQUESTED** [review]
please, could you put a style bold on this line????

File: contact.html (hunk 1 above)
Comment (marcusvbrangel) [reply contact.html:5]: sure!!! no problem!!!

Comment (marcusvbrangel) [reply contact.html:5]: done...

**Review by dijobe — APPROVED** [review]
perfect!!!!! vlw!!!!

**Pull request merged**